slatedb/src/db/builder.rs

```diff
@@ -156,6 +156,43 @@ use crate::stats::StatRegistry;
 use crate::tablestore::TableStore;
 use crate::utils::WatchableOnceCell;
 
+/// Runtime configuration for compaction.
+///
+/// This allows configuring the tokio runtime handle and compaction scheduler
+/// for both standalone compactors and compactors embedded within a Db.
+pub struct CompactorRuntime {
```

Comment:
Nit: bikeshedding, but `Runtime` here makes me a smidge queasy since Tokio's [Runtime](https://docs.rs/tokio/latest/tokio/runtime/index.html) means something else, and is pretty established terminology (it seems). Spitballing `CompactorComponents`/`CompactorContext`? Not sure. Mostly just want to note it.